src/processors/uploadAvatarToIpfs/IPFSUploadError.ts

```diff
@@ -0,0 +1,16 @@
+export class IPFSUploadError extends Error {
+  readonly url: string;
+
+  constructor(message: string, url: string) {
+    super(message);
+
+    // 👇️ because we are extending a built-in class
+    Object.setPrototypeOf(this, IPFSUploadError.prototype);
+
+    this.url = url;
+  }
+
+  getErrorMessage() {
+    return 'Error when uploading file to IPFS: ' + this.message;
+  }
```

Comment:
this function is never used, is it - when the error is caught it will just use the super(message) without the url / prefix text? should this rather use something like `super(getErrorMessage(message, url))` ?

Reply:
You're right; I reworked this function to do that 👌